mongodbatlas/config.go

```diff
@@ -50,15 +47,30 @@ func (c *Config) NewClient(ctx context.Context) (interface{}, diag.Diagnostics)
 		return nil, diag.FromErr(err)
 	}
 
+	clients := &MongoDBClient{
+		Atlas:      atlasClient,
+		PublicKey:  c.PublicKey,
```

Comment:
you could also just reference the config struct instead of copying each of the fields

Reply:
Done, thank you tony !!!